homeassistant/components/homee/strings.json

```diff
@@ -0,0 +1,29 @@
+{
+  "config": {
+    "flow_title": "Homee {name} ({host})",
+    "abort": {
+      "already_configured": "[%key:common::config_flow::abort::already_configured_device%]"
+    },
+    "error": {
+      "cannot_connect": "[%key:common::config_flow::error::cannot_connect%]",
+      "invalid_auth": "[%key:common::config_flow::error::invalid_auth%]",
+      "unknown": "[%key:common::config_flow::error::unknown%]"
+    },
+    "step": {
+      "user": {
+        "title": "Configure homee",
+        "description": "Manually enter the ip address of the homee you want to connect.",
```

Comment:
```suggestion
```
Don't see any reason to add this description as you need to fill in more so its already iomplied